environments/docker.toml

```diff
@@ -584,6 +584,14 @@
         deployed = 8925057
+    [contract.OASIS_ONE]
```

Comment:
There are two Oasis contracts to track, not sure how to differentiate between the two besides "ONE" and "TWO" - totally up for some other ideas!

Reply:
I think the numeric differentiator makes sense 👍 

My only question would be if we potentially want to be more specific - something like `OASIS_MATCHING_MARKET_ONE`. Mostly just not sure if there are other contracts in the Oasis ecosystem we'd want to add later